### PR TITLE
docs(security): add responsible disclosure policy

### PR DESCRIPTION
**Pull request by lirantal**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [ ] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Did you update the typescript typings accordingly (if applicable)?
- [ ] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by papb** [comment]
What about [this part of the readme](https://github.com/sequelize/sequelize/blob/master/README.md#responsible-disclosure)?

Perhaps you should change it too to contain a link to the document you created?

**Comment by lirantal** [comment]
Indeed, I'll push an update.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+If you discover a security vulnerability, please use one of the following means of communications to report it to us:
+
+* Report the security issue to the Node.js Security WG through the 
+[HackerOne program](https://hackerone.com/nodejs-ecosystem) for ecosystem modules on npm, or to
+[Snyk Security Team](https://snyk.io/vulnerability-disclosure). They will help triage the security issue and work with
+all involved parties to remediate and release a fix.
+
+Note that time-frame and processes are subject to each program’s own policy.
+
+* Report the security issue to the project maintainers directly at davidboyne123@hotmail.co.uk. If the report contains
```
Comment (papb) [inline SECURITY.md]: What's this email address? You should link to CONTACT.md here.

Comment (lirantal) [reply SECURITY.md]: good catch, I fixed it.

**Comment by sushantdhiman** [comment]
I have cleaned up `Security.md` a bit :)

File: SECURITY.md [hunk 2]
```diff
@@ -6,39 +6,19 @@ The following table describes the versions of this project that are currently su
...
+## Responsible disclosure policy
 
-A responsible disclosure policy helps protect users of the project from publicly disclosed security vulnerabilities
-without a fix by employing a process where vulnerabilities are first triaged in a private manner, and only publicly
-disclosed after a reasonable time period that allows patching the vulnerability and provides an upgrade path for users.
+At Sequelize, we prioritize security issues and will try to fix them as soon as they are disclosed.
 
-When contacting us directly via email, we will do our best efforts to respond in a reasonable time to resolve the issue.
-When contacting a security program their disclosure policy will provide details on timeframe, processes and paid bounties.
+If you discover a security vulnerability, please reach the project maintainers privately. You can find related information in [CONTACT.md](./CONTACT.md).
```
Comment (lirantal) [inline SECURITY.md]: @sushantdhiman maybe "reach out to the...." instead of "reach the" ?

File: SECURITY.md [hunk 3]
```diff
@@ -6,39 +6,19 @@ The following table describes the versions of this project that are currently su
...
 
-We kindly ask you to refrain from malicious acts that put our users, the project, or any of the project’s team members at
-risk.
+After validating & discussing scope of security vulnerability, we will set a time-frame for patch distribution. This time-frame may vary depending upon the nature of vulnerability.
 
-## Reporting a security issue
+Once effected versions are patched you may report security issue to any Node.js security vulnerability database. A few which we have worked with in past are listed below.
```
Comment (lirantal) [inline SECURITY.md]: @sushantdhiman I think its "affected" versions here

**Comment by lirantal** [comment]
Sure thing. Left some comments on wording, let me know if you'd want a PR.

File: SECURITY.md (hunk 2 above)
Comment (sushantdhiman) [reply SECURITY.md]: de06ac3fd714f7c7eeb10db5774724007928f0c1

File: SECURITY.md (hunk 3 above)
Comment (sushantdhiman) [reply SECURITY.md]: de06ac3fd714f7c7eeb10db5774724007928f0c1

**Comment by sushantdhiman** [comment]
Thanks @lirantal , de06ac3fd714f7c7eeb10db5774724007928f0c1 fixed those issues

**Comment by lirantal** [comment]
@sushantdhiman cool thanks! 👌

**Pull request merged**